cmd/util/cmd/common/flow_client.go

```diff
@@ -47,3 +79,61 @@ func InsecureFlowClient(accessAddress string) (*client.Client, error) {
 
 	return flowClient, nil
 }
+
+// PrepareFlowClientOpts will assemble connection options for the flow client for each access node id
+func PrepareFlowClientOpts(accessNodeIDS []string, insecureAccessAPI bool, snapshot protocol.Snapshot) ([]*FlowClientOpt, error) {
+	flowClientOpts := make([]*FlowClientOpt, 0)
+
+	// convert all IDS to flow.Identifier type, fail early if ID is invalid
+	anIDS := make([]flow.Identifier, 0)
+	for _, anID := range accessNodeIDS {
+		id, err := flow.HexStringToIdentifier(anID)
+		if err != nil {
+			return nil, fmt.Errorf("could not get flow identifer from secured access node id: %s", id)
+		}
+
+		anIDS = append(anIDS, id)
+	}
+
+	identities, err := snapshot.Identities(filter.HasNodeID(anIDS...))
+	if err != nil {
+		return nil, fmt.Errorf("failed get identities access node IDs from snapshot: %v", accessNodeIDS)
+	}
+
+	// make sure we have identities for all the access node IDs provided
+	if len(identities) != len(accessNodeIDS) {
+		return nil, fmt.Errorf("failed to get identity for all the access node IDs provided: %v, got %s", accessNodeIDS, identities)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to get identity for all the access node IDs provided: %v, got %v", accessNodeIDS, identities.NodeIDs())
```

so the expected and actual values are in the same format

Reply:
https://github.com/onflow/flow-go/pull/1395/commits/63038258db5767cf410d7b0178d2c13bf2cf1e35